### PR TITLE
[FM-6956] Removal of unsupported OS from apt

### PR DESCRIPTION
**Pull request by david22swan**

### CODE REVIEW

**Comment by david22swan** [comment]
https://github.com/puppetlabs/ci-job-configs/pull/4540

**Comment by david22swan** [comment]
failure is a pre-existing bug.

File: spec/classes/apt_spec.rb [hunk 1]
```diff
@@ -312,9 +312,9 @@
   context 'with settings defined on valid osfamily' do
     let :facts do
       {
-        os: { family: 'Debian', name: 'Ubuntu', release: { major: '12', full: '12.04.5' } },
+        os: { family: 'Debian', name: 'Ubuntu', release: { major: '16', full: '16.04' } },
         osfamily: 'Debian',
-        lsbdistcodename: 'precise',
+        lsbdistcodename: 'xenian',
```
Comment (HelenCampbell) [inline spec/classes/apt_spec.rb]: Typos in all of these, *xenial

Comment (david22swan) [reply spec/classes/apt_spec.rb]: Fixed

**Comment by HelenCampbell** [comment]
Merging due to failures being a separate issue

**Pull request merged**